kustomize/commands/internal/util/util_test.go

```diff
@@ -52,68 +42,69 @@ func TestConvertSliceToMap(t *testing.T) {
 	expected["g"] = "h:k"
 
 	result, err := ConvertSliceToMap(args, "annotation")
-	if err != nil {
-		t.Errorf("unexpected error: %v", err.Error())
-	}
-
-	eq := reflect.DeepEqual(expected, result)
-	if !eq {
-		t.Errorf("Converted map does not match expected, expected: %v, result: %v\n", expected, result)
-	}
+	require.NoError(t, err)
+	require.Equal(t, expected, result)
 }
 
 func TestGlobPatternsWithLoaderRemoteFile(t *testing.T) {
+	req := require.New(t)
+
 	fSys := filesys.MakeFsInMemory()
-	fSys.Create("test.yml")
+	_, err := fSys.Create("test.yml")
+	req.NoError(err)
+
 	httpPath := "https://example.com/example.yaml"
-	ldr := fakeLoader{
-		path: httpPath,
+	ldr := fakeRemoteLoader{
+		files: map[string]struct{}{httpPath: {}},
 	}
+	expected := [2]string{httpPath, "/test.yml"}
 
 	// test load remote file
-	resources, err := GlobPatternsWithLoader(fSys, ldr, []string{httpPath})
-	if err != nil {
-		t.Fatalf("unexpected load error: %v", err)
-	}
-	if len(resources) != 1 || resources[0] != httpPath {
-		t.Fatalf("incorrect resources: %v", resources)
-	}
+	resources, err := globPatternsWithLoader(fSys, ldr, []string{httpPath})
+	req.NoError(err)
```

Comment:
There seem to be some test changes here that are relevant to the `Loader` change, and some that aren't. Could you move the extra test fixes to their own PR?

Reply:
Done. Only the extra fixes remain in this PR. The actual `Loader` change will go in #4652